docs/specifications/tests/DNSSEC-TP/dnssec05.md

```diff
@@ -5,20 +5,24 @@
 
 ## Objective
 
-A domain name (zone) should only use DNSKEY algorithms that are specified 
-by IANA to be used for a DNSKEY. A public domain name (zone) should not
-use private algorithms. The IANA registry of [DNSSEC Algorithm Numbers]
-specifies which algorithms to use.
+A domain name (zone) should only use DNSKEY algorithms that are specified
+by [RFC 8624], section 3.1 and the [IANA registry] of *DNSSEC Algorithm
+Numbers* to be used for DNSSEC signing. A public domain name (zone) should not use
+private algorithms.
```

Comment:
I am not sure is it a good idea to have both the RFC and the registry as reference. The registry will be updated, whereas the RFC will not be..

Reply:
I agree that it is complex, but I think we have to meet the stricter specification in the RFC at the same time as we should keep a reference to the registry. There algorithms that the registry covers that are not mentioned in the RFC.

When the registry is updated, there will be a simple update if they just adapt to the RFC (which I think they will) or else we have to decide how we should balance.

Independent of what we do, we have to follow the change of RFC and the update of the registry.